domain/network/service/subnet.go

```diff
@@ -20,11 +19,11 @@ func (s *Service) AddSubnet(ctx context.Context, args network.SubnetInfo) (netwo
 	defer span.End()
 
 	if args.ID == "" {
-		uuid, err := uuid.NewV7()
```

Comment:
This was picked over our uuid implementation. There was a reason, although that escapes me.